app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -131,9 +133,9 @@ def server_groups(feed, _domain)
       end
     end
 
-    def child_resources(resource_path)
+    def child_resources(resource_path, recursive = false)
```

Comment:
It's because of hierarchy in inventory, JMS are kept in recursive children, where at first level (non recursive) is only messaging server (this is how it's actually named in inventory, it's just resource with child entities), and queues / topics are located under this entity.